README.md

```diff
@@ -105,7 +105,9 @@ cd $GOPATH/src/github.com/skycoin/skycoin
 make ARGS="--launch-browser=false -data-dir=/custom/path" run
 ```
 
-### Docker image
+### Docker images
```

Comment:
The anchor in the top of the README is `#docker-image`, which should be `#docker-images`.

Reply:
Done